test/src/unit-capi-array_schema.cc

```diff
@@ -2771,3 +2771,540 @@ TEST_CASE_METHOD(
   delete_array(array_name);
   remove_temp_dir(local_fs.file_prefix() + local_fs.temp_dir());
 }
+
+TEST_CASE_METHOD(
+    ArraySchemaFx,
+    "C API: Test add and drop attributes at timestamp",
+    "[capi][array-schema]") {
```

Comment:
This is unrelated, but I've (somehow) just realized that we aren't consistently using `-` vs. `_` in our unit tests. For example, this test can be run with `./tiledb_unit "[capi][array-schema]"`, whereas the `FragmentInfo` test uses underlines: `"[capi][fragment_info]"` - this should be consistent for users.

Reply:
I'm going to leave as-is in this PR since the rest of the unit tests in here use `[array-schema]`, but I think this is a good observation and definitely would be worth auditing the unit tests to be consistent about using either dashes or underscores.